### PR TITLE
Interactive plots in documentation

### LINKED ISSUE #260: Add 2D plot for emissions and footprints for tutorials

### PR DESCRIPTION
**Pull request by SutarPrasad**
* **Summary of changes** (Bug fix, feature, docs update, ...)
     Updated Make command to copy the HTML files from source/images to build/Exploring data/images
     Added the HTML versions of the surface plots present in the current tutorial documentation. User pio.write_html to             generate the HTML versions of the plots
     
    The cp command was added because the docs were not able to access the files from source directly.

     Pending:  footprints and emissions plot

* **Please check if the PR fulfills these requirements**

- [x] All code checks passing - `black --line-length 110` run over code, `mypy` and `flake8` not showing any errors
- [x] Documentation and tutorials updated/added
- [x] Added an entry in the latest `CHANGELOG.md` file if fixing a bug or adding a new feature

### CODE REVIEW

**Review by rt17603 — COMMENTED** [review]
This looks great and happy for this to go in but this won't close #260 as that issue was around creation of new plotting routines for lat-lon data (2D data) rather than showing the 1D timeseries plots we had within the online documentation. Still a very useful PR though.

File: CHANGELOG.md
Comment (rt17603) [inline CHANGELOG.md:14]:
```suggestion
- Added interactive timeseries plots in Search and Plotting tutorial. - [PR #953](https://github.com/openghg/openghg/pull/953)
```

**Comment by rt17603** [comment]
Happy to approve once merge conflicts in have been resolved on this.

File: CHANGELOG.md
Comment (rt17603) [reply CHANGELOG.md:14]: Once this change has been made I can add an Approve for this PR.

**Comment by SutarPrasad** [comment]
> Happy to approve once merge conflicts in have been resolved on this.

its done now

**Review by rt17603 — APPROVED** [review]
Looks good!

**Pull request merged**
